code/game/objects/structures/crates_lockers/closets/secure/secure_closets.dm

```diff
@@ -87,3 +87,42 @@
 		togglelock(usr)
 	else
 		to_chat(usr, SPAN_WARNING("This mob type can't use this verb."))
+
+// Marooning Equipment
+/obj/structure/closet/secure_closet/marooning_equipment
+	name = "marooning equipment locker"
+	icon_state = "maroon"
+	req_one_access = list(access_security, access_heads) // Marooned personnel would likely be marooned by security and/or command.
+
+/obj/structure/closet/secure_closet/marooning_equipment/fill()
+	new /obj/item/clothing/mask/breath(src)
+	new /obj/item/clothing/under/color/yellow(src)
+	new /obj/item/clothing/shoes/workboots/grey(src)
+	new /obj/item/clothing/head/helmet/space/emergency/marooning_equipment(src)
+	new /obj/item/clothing/suit/space/emergency/marooning_equipment(src)
+	new /obj/item/tank/oxygen/marooning_equipment(src)
+	new /obj/item/storage/backpack/duffel/marooning_equipment(src)
+
+/obj/item/storage/backpack/duffel/marooning_equipment
+	name = "marooning equipment duffel bag"
+	desc = "A duffel bag full of marooning equipment."
+	starts_with = list(
+		// Weapon
+		/obj/item/gun/energy/mousegun/xenofauna = 1,
+
+		// Equipment
+		/obj/item/crowbar/rescue_axe/red = 1,
```

Comment:
I am really unsure about these two.
The rescue axe makes it relatively easy to break into areas (which you definitely don't want)
The pest gun is a "why bother" the point of marooning is to get rid of someone without "making your hands dirty".
Giving them more than the bare essentials is imho wasted effort (by the scc).

Reply:
leaving someone to without proper tools is manslaughter.

Reply:
The entire point there is that its a capital punishment similar to borging (or execution in some jurisdictions).
Given that marooning is performed on terrorists, mutineers, murderers and corporate spies, the SCC would want to avoid them coming back to the ship alive to take their revenge.

Reply:
fair. will remove.